src/libraries/Microsoft.Extensions.Http/src/ValueStopwatch.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Extensions.Internal
 {
-    internal struct ValueStopwatch
+    internal readonly struct ValueStopwatch
```

Comment:
Unrelated to your PR, but this looks like it could be replaced or simplified with `Stopwatch.GetElapsedTime` since we exposed that in .NET 7.

Reply:
Yes, but this library targets netstandard2.0 and net462 as well.  I had to exempt it when I added GetElapsedTime and rolled out its usage.